### ISSUE TITLE
Every Pixel Matters: Center-aware Feature Alignment for Domain Adaptive Object Detector

### ISSUE THREAD

**Issue by chaerinkong** [reyllama/paper-reviews#126]
## TL; DR

- Overcome the downfall of object-level and instance-level feature alignment through *center-aware* feature alignment that operates on individual pixel level (*hence Every pixel matters*)

## Motivation

- Image-level feature alignment: not so effective with object detectors
- Instance-level feature alignment: region proposal in the target domain can be noisy

## Previous Works

- Object Detection: Fast-RCNN, Faster-RCNN, FCOS
- Unsupervised Domain Adaptation: Image level, Instance Level...

## Method

![image](https://user-images.githubusercontent.com/46308407/144001941-1e06b0b4-f2c5-4d20-a3b6-caaecfb5ec06.png)

### (1) Global Alignment

![image](https://user-images.githubusercontent.com/46308407/144002186-8e977d6b-f5fb-4bc7-af22-3b4a2cba3e2b.png)

### (2) Center-aware Alignment

![image](https://user-images.githubusercontent.com/46308407/144002211-eb970224-5caf-4e75-b7bf-d6819f4cc722.png)

- Network predicts per-pixel centerness score, which is used to scale alignment loss

![image](https://user-images.githubusercontent.com/46308407/144002324-3fd7d559-11bc-42e0-8539-46e27c533afa.png)
![image](https://user-images.githubusercontent.com/46308407/144002347-5705cdc5-8cfd-405b-95f3-0eb6deced354.png)

### (3) Training

![image](https://user-images.githubusercontent.com/46308407/144002388-2286461b-5294-42f1-9b1c-ff2fe98f6145.png)
![image](https://user-images.githubusercontent.com/46308407/144002402-0f396c75-6d6d-46a3-9cfe-57864a4eefb3.png)

### (1) Weather Adaptation

![image](https://user-images.githubusercontent.com/46308407/144002525-8e52c9ed-bb82-4676-8722-6905a99124dc.png)

### (2) Synthetic-to-Real / Cross-camera Adaptation

![image](https://user-images.githubusercontent.com/46308407/144002570-55738cb2-4ef2-42a6-b6f9-7eaf7f3954cf.png)

![image](https://user-images.githubusercontent.com/46308407/144002647-eade81ef-5f45-4a1a-8c9e-9c1006895c21.png)
![image](https://user-images.githubusercontent.com/46308407/144002675-48f645cf-55ea-4d65-b5a1-397d80e0d180.png)

## Comments

- Interesting research topic with clear practical implications.
